deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -96,7 +96,7 @@ spec:
                   Custom container image registry server that will be used instead of
                   index.docker.io to download Docker images. This may include the repository as well.
                   (Example: myregistry.net:5443 or myregistry.com/myrepository)
-              preserveFullCustomImageRegistry:
+              preserveFullCustomRegistryPath:
```

Comment:
To be consistent with `customImageRegistry` can we call this `preserveFullCustomImageRegistry` or `preserveFullCustomImageRegistryPath`? Not sure if we will have a different registry than image registry in future, but it is better to be safe as this is a user facing field.
I know this involves more changes in the code, but better get this done once.

Reply:
Sounds good, changed.